doc/motion_planning_pipeline/motion_planning_pipeline_tutorial.rst

```diff
@@ -15,7 +15,11 @@ If you haven't already done so, make sure you've completed the steps in `Getting
 
 Running the Code
 ----------------
-Roslaunch the launch file to run the code directly from moveit_tutorials: ::
+Open two shells. In the first shell start RViz and wait for everything to finish loading: ::
```

Comment:
Instead of requiring the user to start two launch files, I suggest to include `demo.launch` in the `motion_planning_pipeline_tutorial.launch`

Reply:
We do this for virtually every other tutorial demo. A change like that can be made repo wide separately

Reply:
I believe I'm the one who advocated keeping them separate in the past. I want users to learn a smooth development pattern, which IMHO is to always keep Rviz loaded rather than having to resize and move it everytime. And wait for it to load everytime.

Reply:
How is having rviz as a separate rosrun/roslaunch process related to running the tutorial?
I do see the advantage of having an rviz run all the time. But demo.launch already starts rviz!
But I don't see a benefit in running two separate launch files for a single demo.

Reply:
Besides the advantages above, its also nice to isolate the output of the tutorial (useful console output) from rviz, move_group, etc. 

demo.launch starts rviz and move_group, yes

the second roslaunch is the example code we want the new user to focus on and learn. 

its cleaner when separate.

Reply:
If there is relevant output, I agree.